src/config_types.ts

```diff
@@ -105,6 +106,7 @@ function contextIterator(): u.ListIterator<any, Context> {
             cluster: elt.context.cluster,
             name: elt.name,
             user: (elt.context.user ? elt.context.user : undefined),
+            namespace: elt.context.namespace || undefined,
```

Comment:
Could we standise on one of the above to formats? Not sure which one is better though.

Reply:
`||` syntax is fine with me, and agree standardization is good.